### PR TITLE
white label data added

### PR DESCRIPTION
**Pull request by tanujgupta**

### CODE REVIEW

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt [hunk 1]
```diff
@@ -139,30 +145,52 @@ class OpenloginFlutterPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, P
...
+
+      else -> Provider.GOOGLE
     }
   }
 
-  private fun getOpenLoginNetwork(network : String) : OpenLogin.Network {
-    if (network.equals("mainnet", true)) return OpenLogin.Network.MAINNET
-    else return OpenLogin.Network.TESTNET
+  private fun getOpenLoginNetwork(network : String) : Web3Auth.Network {
+    if (network.equals("mainnet", true)) return Web3Auth.Network.MAINNET
```
Comment (chaitanyapotti) [inline android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:170]: pls add cyan network support too
default is mainnet
so, if no network is specified, it should be mainnet

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt [hunk 2]
```diff
@@ -139,30 +145,52 @@ class OpenloginFlutterPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, P
...
+    else return Web3Auth.Network.TESTNET
+  }
+
+  private fun mapWhiteLabelData(call: MethodCall) : WhiteLabelData? {
+    val wlName : String? = call.argument("wl_name")
```
Comment (chaitanyapotti) [inline android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:175]: why are the arguments different.?

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt [hunk 3]
```diff
@@ -187,7 +215,7 @@ class OpenloginFlutterPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, P
 
     return LoginParams(getOpenLoginProvider(provider),
             appState = appState,
-            reLogin = reLogin,
+            relogin = reLogin,
```
Comment (chaitanyapotti) [inline android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:218]: pls make the user exposed sdk same as web. meaning change case of relogin

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt (hunk 1 above)
Comment (tanujgupta) [reply android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:170]: done

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt (hunk 2 above)
Comment (tanujgupta) [reply android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:175]: done

File: android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt (hunk 3 above)
Comment (tanujgupta) [reply android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt:218]: done

**Pull request merged**